python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -260,6 +260,8 @@ def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
         desc = f"- {self.op.name}: {active} active, {queued} queued"
+        if self.op._in_task_submission_backpressure:
```

Comment:
besides "task submission backpressure", we should also include "task output backpresure", I.E., when this is 0 https://github.com/raulchen/ray/blob/b775a604ce084b1242930091686af6f751dbba32/python/ray/data/_internal/execution/streaming_executor_state.py#L393